site2/docs/administration-pulsar-shell.md

```diff
@@ -0,0 +1,174 @@
+---
+id: administration-pulsar-shell
+title: Pulsar Shell
+sidebar_label: "Pulsar Shell"
+---
+
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+````
+
+
+Pulsar shell is a fast and flexible shell for Pulsar cluster management, messaging, and more.
+It's great for quickly switching between different clusters, and can modify cluster or tenant configurations in an instant.
+
+
+# Features
```

Comment:
```suggestion
## Features
```

The content inside a topic starts with Heading2.
Does it make sense to use "Use case" as the title?